index.js

```diff
@@ -133,9 +132,10 @@ else if (program.hub) {
     console.log("------ Hub enumerate devices for %j".header, program.hub);
 
     var fileName = helpers.createOnboardingFileName(program.hub);
-    helpers.readFile(fileName, "Please complete onboarding -o").then(data => { 
-        var deviceInfo = JSON.parse(data).authInfo;
-        translatorCli.getProperty(program.hub, deviceInfo, 'getPlatforms').then(info => {
+    helpers.readFile(fileName, "Please complete onboarding -o").then(data => {
+        var configInfo = JSON.parse(data);
+        var deviceInfo = configInfo.authInfo;
+        translatorCli.getProperty(configInfo.translatorPackageName, deviceInfo, 'getPlatforms').then(info => {
             helpers.logObject(info);
```

Comment:
So maybe use helpers.logObject() consistently instead of direct console.log's for a start? And then tackle migrating helpers.logObject() using Opent2t.Logger with a later change